common/rpc/interceptor/namespace_validator.go

```diff
@@ -71,10 +71,30 @@ var (
 	defaultAllowedNamespaceStates = []enumspb.NamespaceState{enumspb.NAMESPACE_STATE_REGISTERED, enumspb.NAMESPACE_STATE_DEPRECATED}
 
 	allowedMethodsDuringHandover = map[string]struct{}{
-		"UpdateNamespace":                  {},
-		"GetReplicationMessages":           {},
-		"ReplicateEventsV2":                {},
-		"GetWorkflowExecutionRawHistoryV2": {},
+		"DescribeNamespace":                  {},
+		"UpdateNamespace":                    {},
+		"GetReplicationMessages":             {},
+		"ReplicateEventsV2":                  {},
+		"GetWorkflowExecutionRawHistoryV2":   {},
```

Comment:
`GetWorkflowExecutionRawHistory` api as well? It is a new api built to retrieve events inclusive/inclusive.

Reply:
Have we full move to V2 API?

Reply:
Not yet. The plan was to get rid of V2 api (exclusive/exclusive) and keep inclusive/inclusive api.